docs/rancher/cloud-provider.md

```diff
@@ -62,25 +62,88 @@ When spinning up an RKE2 cluster using the Harvester node driver, select the `Ha
-   ```
+    ```bash
+    curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace>
+    ```
 
   :::note
 
-  The `generate_addon.sh` script depends on `kubectl` and `jq` to operate the Harvester cluster.
+    The script depends on `kubectl` and `jq` when operating the Harvester cluster, and functions only when given access to the `Harvester Cluster` kubeconfig file.
 
-  The script needs access to the `Harvester Cluster` kubeconfig to work. You can find the `kubeconfig` file from one of the Harvester management nodes in the `/etc/rancher/rke2/rke2.yaml` path.
+    You can find the `kubeconfig` file in one of the Harvester management nodes in the `/etc/rancher/rke2/rke2.yaml` path. The server IP must be replaced with the VIP address.
+
+  Example of content:
+
+    ```yaml
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: <redacted>
+        server: https://127.0.0.1:6443
+      name: default
+    # ...
+    ```
 
-  The namespace needs to be the namespace in which the guest cluster will be created.
+    You must specify the namespace in which the guest cluster will be created.
 
   :::
 
+    Example of output:
 
-2. Configure the **Cloud Provider** to `Harvester` and select **Create** to spin up the cluster.
-     ![](/img/v1.2/rancher/create-custom-rke2.png)
+    ```yaml
+    ########## cloud config ############
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: <CACERT>
+        server: https://HARVESTER-ENDPOINT/k8s/clusters/local
+      name: local
+    contexts:
+    - context:
+        cluster: local
+        namespace: default
+        user: harvester-cloud-provider-default-local
+      name: harvester-cloud-provider-default-local
+    current-context: harvester-cloud-provider-default-local
+    kind: Config
+    preferences: {}
+    users:
+    - name: harvester-cloud-provider-default-local
+      user:
+        token: <TOKEN>
+
+    ########## cloud-init user data ############
+    write_files:
+    - encoding: b64
+      content: <CONTENT>
+      owner: root:root
+      path: /etc/kubernetes/cloud-config
+      permissions: '0644'
+    ```
+
+1. Create a VM in the Harvester cluster with the following settings:
+
+  - **Basics** tab: The minimum requirements are 2 CPUs and 4 GiB of RAM. The required disk space depends on the VM image.
+
+    ![](/img/v1.3/rancher/custom-cluster-vm-cpu-and-ram.png)
+
+  - **Networks** tab: Specify a network name with the format `nic-<number>`.
+
+    ![](/img/v1.3/rancher/custom-cluster-vm-network.png)
+
+  - **Advanced Options** tab: Copy and paste the content of the **Cloud Config User Data** screen.
+
+    ![](/img/v1.3/rancher/custom-cluster-vm-user-data.png)
+
+1. On the **Basics** tab of the **Cluster Configuration** screen, select **Harvester** as the **Cloud Provider** and then select **Create** to spin up the cluster.
```

Comment:
In the [doc preview](https://66863d5f42fbf17e30f4e778--harvester-preview.netlify.app/v1.4/rancher/cloud-provider#deploying-to-the-rke2-custom-cluster-experimental), the ordered list starts with `1.` again. I'm not sure if the intention is to continue with `3.`. And we should mention here the user should go to the Rancher's GUI, but not Harvester's GUI anymore.

Please also add the doc source to v1.3/v1.2 versions if needed (it's currently only in the dev version).

Reply:
Fixed it and also migrated the document to v1.3/v1.2. Thanks.